ElementX/Sources/Screens/HomeScreen/View/HomeScreenRecoveryKeyConfirmationBanner.swift

```diff
@@ -68,6 +68,9 @@ struct HomeScreenRecoveryKeyConfirmationBanner: View {
                         Image(systemName: "xmark")
                             .foregroundColor(.compound.iconSecondary)
                             .frame(width: 12, height: 12)
+                            // Increase hit area
+                            .padding(.leading, 5)
+                            .padding(.bottom, 2)
                     }
```

Comment:
Should we not also then

```suggestion
                    }
                    .padding(.leading, -5)
                    .padding(.bottom, -2)
```
here so that the layout still matches the designs?

Reply:
the preview tests did not fail, also I am not sure if reducing the padding externally of leading and bottom would be the same.

Reply:
It would probably only be noticeable when the screen/localisation would put the button close to the label.

Reply:
Okay I found out that the hit region issues are not being flagged anymore, probably they were caused by the dynamic type scaling not working prior to the fix, because now if I run the hit region tests I don't get any issue!